src/main/java/jewellery/inventory/service/ProductService.java

```diff
@@ -59,6 +57,16 @@ public ProductResponseDto getProduct(UUID id) {
     return productMapper.mapToProductResponseDto(product);
   }
 
+  public Product returnProductByID(UUID id) {
+    return productRepository.findById(id).orElseThrow(() -> new ProductNotFoundException(id));
+  }
```

Comment:
We have a method named getProduct that has almost the same signature. Please, rename the other one to `getProductResponse` and rename this one to `getProduct`